specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/examples/MultipleModernInvoiceDownload.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2020-05-01",
+    "billingAccountName": "{billingAccountName}",
+    "downloadUrls": [
+      "https://management.azure.com/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoices/{invoiceName}/download?downloadToken={downloadToken}&useCache=True&api-version=2020-05-01",
```

Comment:
how does a user craft these URIs?  Why not just take the resource ID of the relevant resource as input instead of a full URI to ARM?  Its very rare to see a full ARM URI (with the host) taken as input to an API.

Reply:
The user does not need to craft these URIs, they get the URIs from the ListInvoices GET request in the format shown below:

```json
{
  ...
  "documents": [
    {
      "kind": "Invoice",
      "url": "string",
      "source": "DRS"
    }
  ]
  ...
} 
```

The decision we had to make was whether we have the user parse the URL and give us the invoiceName and downloadToken of the relevant resource, or to have them pass the entire resource URL back to us and we parse it instead.

Thoughts?